### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jpp9**
First pass of general SHL guidance, based on current spec.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,102 @@
...
+| [SMART Health Link Management](Generic_Mobile_UX_Link_Manage.png) | Sharing and management screen for a particular SMART Health Link, allowing the user to view, copy, and share the link as well as view and manage access |
+| [SMART Health Link QR](Generic_Mobile_UX_SHL_QR.png) | QR code display screen for a particular SMART Health Link |
+| [SMART Health Link Access Log](Generic_Mobile_UX_Access_Log.png) | Access log screen for a particular SMART Health Link |
+
+## Key UX Elements
+
+### SMART Health Link Setup
+Provide users with a clear overview of the SMART Health Link they plan to create (depicted in [SMART Health Link Setup](Generic_Mobile_UX_Link_Set-up.png)). Include,
+
+* Label for the link (e.g. “Jane’s Complete Immunizations”)
```
Comment (jmandel) [inline README.md:25]: Should we clarify that this is the same as the label embedded in the SHL itself? In other words, the value here needs to be a <80 char string that will be visible to anyone who obtains the SHL, **even before they demonstrate knowledge of the passcode**? Or are we imagining this as a private label in the management UX distinct from what, if anything, is embedded in the SHL?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,102 @@
...
+* Expiration information
+* Passcode
+* Description or list of data included in the SMART Health Link, or the action the link initiates, if not otherwise obvious in the label or UX
+
+Be sure that users understand that by activating this link, all of the referenced data will be available to anyone with access to the link.
```
Comment (jmandel) [inline README.md:30]: Anyone who has the link and knows the passcode, assuming there is one

File: README.md [hunk 3]
```diff
@@ -1,6 +1,102 @@
...
+
+### SMART Health Link Management
+For active SMART Health Links, provide users with tools to manage the key details of that link (depicted in [SMART Health Link Management](Generic_Mobile_UX_Link_Manage.png)). Users should be able to view or configure,
+
+* The Label for the link (e.g. “Jane’s Complete Immunizations”)
+* Expiration date
+* Passcode
+* The link URI
```
Comment (jmandel) [inline README.md:38]: What about the URI should be configurable?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,102 @@
...
+Be sure that users understand that by sharing this link, all of the referenced data will be available to anyone with access to the link.
+
+### SMART Health Link QR Code
+For active SMART Health Links, provide users with tools to share as a QR code as they see fit (depicted in [SMART Health Link QR](Generic_Mobile_UX_SHL_QR.png)).
+
+QR codes should be presented in close proximity to the Label for the SMART Health Link so it is clear what is being shared. Include any other key information needed to identify the link and understand the contents, such as patient name or date of birth.
+
+QR Codes should be rendered in black on a white background only to maximize contrast. The QR code should have a clear white border.
+
+Embed the SMART logo in the QR code to increase recognition by users and verifiers.
```
Comment (jmandel) [inline README.md:61]: We currently have some language to the effect at https://docs.smarthealthit.org/smart-health-links/spec#3-data-sharer-transmits-a-shlink

Should we reference or reconcile?

**Review by jmandel — COMMENTED** [review]
This is awesome! I heard a few tiny comments but happy to merge anytime

File: README.md (hunk 1 above)
Comment (jpp9) [reply README.md:25]: i wonder if a little vocab section would be useful?

**Comment by jpp9** [comment]
great comments. addressed each of them. i'll think about adding a vocab section to make things like label and recipient more clear.

**Pull request merged**